helm/exim/templates/statefulsetOrdeployment.yaml

```diff
@@ -68,13 +68,17 @@ spec:
           - name: log-volume
             mountPath: /var/log/exim
       {{- end }}
-      {{- if .Values.mtail.enabled }}
-      - name: mtail
-        image: '{{ .Values.mtail.image.repository }}:{{ .Values.mtail.image.tag }}'
+      {{- if .Values.eximexporter.enabled }}
+      - name: eximexporter
+        image: '{{ .Values.eximexporter.image.repository }}:{{ .Values.eximexporter.image.tag }}'
         imagePullPolicy: IfNotPresent
         volumeMounts:
           - name: log-volume
             mountPath: /var/log/exim
+          {{- if eq .Values.workload "StatefulSet" }}
```

Comment:
Why are you mounting the PV everywhere? It's a ReadWriteOnce, you cannot do that.

Reply:
can't the pvc access mode change to ROX as we have a requirement here ?